typescript/types.ts

```diff
@@ -192,8 +192,12 @@ export type ExecuteResult = {
   /**
    * The result of executing the prompt.
    */
-  data: JSONValue;
-
+  data:
+    | JSONValue
+    | {
```

Comment:
The thing I'm not sure of here is if we should have this data associated with an Attachment so we can know its mimetype. Even if we have file_uri, we still need to know what to render it as (image, video, audio, etc.)